Rubberduck.Core/Formatters/InspectionResultFormatter.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Rubberduck.Common;
+using Rubberduck.Parsing.Inspections.Abstract;
+using Rubberduck.Inspections.Abstract;
+using System.IO;
+using Rubberduck.Resources.Inspections;
+
+namespace Rubberduck.Formatters
+{
+    public class InspectionResultFormatter : IExportable
+    {
+        private readonly IInspectionResult _inspectionResult;
+
+        public InspectionResultFormatter(IInspectionResult inspectionResult)
+        {
+            _inspectionResult = inspectionResult;
+        }
+
+        public object[] ToArray()
+        {
+            var module = _inspectionResult.QualifiedSelection.QualifiedName;
+            return new object[]
+            {
+                _inspectionResult.Inspection.Severity.ToString(),
+                module.ProjectName,
+                module.ComponentName,
+                _inspectionResult.Description,
+                _inspectionResult.QualifiedSelection.Selection.StartLine,
+                _inspectionResult.QualifiedSelection.Selection.StartColumn
+            };
+        }
+
+        /// <summary>
+        /// WARNING: This property can have side effects. It can change the ActiveVBProject if the result has a null Declaration, 
+        /// which causes a flicker in the VBE. This should only be called if it is *absolutely* necessary.
+        /// </summary>
```

Comment:
This feels kind of weird -- why would this need to access the VBIDE? Surely we should be reading off the `QualifiedModuleName.ProjectName` instead of accessing `ProjectDisplayName`?

Reply:
This is not in the scope of this PR.

I plan to have a look at it when I remove the project wrapper reference from the project declaration. 

The problem why the project name is not sufficient is that we want the filename as well. Otherwise, the location is not unique. For user projects, it is not part of the QMNs because it is not always disclosed by the VBE API, e.g. if the file is unsaved.